consensus/core/src/dag_state.rs

```diff
@@ -728,6 +728,23 @@ impl DagState {
         self.last_committed_rounds.clone()
     }
 
+    /// The latest Garbage Collection (GC) round that is calculated based on the latest committed leader round. When GC is disabled that will return the genesis round.
+    pub(crate) fn gc_round(&self) -> Round {
```

Comment:
Can you mention that GC round is the lowest round where blocks get included into the DAG and commits.

Reply:
If we use the 2nd definition, semantically genesis round blocks will never get included into the DAG, which may be ok.

Reply:
Sure I can add the definition as commend. 

> If we use the 2nd definition, which seems to the case below, genesis round blocks will not get included into the DAG, which may be ok

I believe that's fine and inline with what we do today - skipping genesis blocks. During the linearization we filter all ancestor blocks for which `last_committed_rounds[ancestor.author] < ancestor.round`.